controller/place.js

```diff
@@ -24,9 +20,7 @@ function setup( config, backend ){
 
     logger.debug( '[ES req]', query );
 
-    service.mget( backend, query, function( err, docs ) {
-      console.log('err:' + err);
```

Comment:
did you intend to remove this error logging?

it seems a bit weird to log every time, since I'm guessing for a vast majority of times it would be `null` anyway